### PR TITLE
Adds ability to check if an account is a test or production account.

### PR DESCRIPTION
**Pull request by magnus21**

### CODE REVIEW

File: src/main/java/com/qapital/plaid/client/DefaultPlaidUserClient.java [hunk 1]
```diff
@@ -50,6 +53,11 @@ public String getAccessToken() {
     }
 
     @Override
+    public Boolean isProductionAccount() {
```
Comment (blockmar) [inline src/main/java/com/qapital/plaid/client/DefaultPlaidUserClient.java]: Use a small `boolean` please. Return value can never be `null`.

**Review by blockmar — CHANGES_REQUESTED** [review]
Just one small comment

File: src/main/java/com/qapital/plaid/client/DefaultPlaidUserClient.java (hunk 1 above)
Comment (magnus21) [reply src/main/java/com/qapital/plaid/client/DefaultPlaidUserClient.java]: Sure, you prefer primitive types when possible?

Comment (blockmar) [reply src/main/java/com/qapital/plaid/client/DefaultPlaidUserClient.java]: Yes. Smaller memory footprint. But most importantly no null pointer issues to worry about. If I saw this in code I could feel the need to check if the reply could ever be `null`.

**Review by blockmar — APPROVED** [review]

**Pull request merged**